### PR TITLE
Be explicit with the transitioned properties

### PR DESCRIPTION
**Pull request by XhmikosR**
Needs testing to make sure it's fine.

### CODE REVIEW

**Comment by thomaspark** [comment]
They're looking good to me.

File: dist/materia/_bootswatch.scss [hunk 1]
```diff
@@ -354,10 +354,11 @@ select.form-control {
   &:checked {
     ~ .custom-control-label::before {
       background: $white;
+      transition: border-color .24s;
```
Comment (XhmikosR) [inline dist/materia/_bootswatch.scss:357]: I'm not sure about this change TBH. Can you verify if this is needed? Maybe the next change needs some tweaking but I generally tried to keep things simple.

Comment (thomaspark) [reply dist/materia/_bootswatch.scss:357]: I checked (no pun intended) and it's needed to set the transition duration to the same `::after`. The default being .15s.

Comment (XhmikosR) [reply dist/materia/_bootswatch.scss:357]: If you triple checked and everything looks the same, feel free to merge :)

**Pull request merged**